styles/06-components/brand-theme.css

```diff
@@ -1,6 +1,7 @@
 body {
   color: var(--color-text);
   background-color: var(--color-background);
+  min-height: 100vh;
 }
 
 .nav-main {
```

Comment:
forgot to remove this probably, it has no CSS closing bracket too

Reply:
hi, same as above, please press the ... on the left